src/govuk/helpers/_links.scss

```
@@ -103,12 +106,13 @@
   }
 }
 
-/// Error link style mixin
+/// Error link styles
 ///
-/// Provides the error unvisited, visited, hover and active states for links.
+/// Makes links use the error colour, darkening when the link is hovered over or
```

```suggestion
/// Makes links use the error colour. Links will darken when the link is hovered over or
```

This might be better as two sentences.

---

Could you make this more active too? For example: "A link will darken if it's active or a user is hovering over it."